green_cli/common.py

```diff
@@ -346,12 +346,13 @@ def getreceiveaddress(session, details):
 
 @green.command()
 @click.option('--subaccount', default=0, expose_value=False, callback=details_json)
-@click.option('--last-pointer', default=0, expose_value=False, callback=details_json)
+@click.option('--last-pointer', default=None, expose_value=False, callback=details_json)
```

Comment:
I think the `default=None` can be omitted, but add `type=int` so that the help displays correctly

Reply:
Done